README.md

```diff
@@ -7,6 +7,17 @@ A python implementation of the UA Parser (https://github.com/ua-parser, formerly
 
 ##Installing
 
+### Install via pip
+First, check that you've got the correct dependencies. For example, on Ubuntu:
+```
+$ apt-get install libyaml libyaml-dev python-dev
+```
+Then, just run:
+```
+$ pip install pyyaml ua-parser user-agents
```

Comment:
You should only need `$ pip install ua-parser` here.

`pyyaml` is a direct dependency and will be pulled in with it. And not sure what `user-agents` is, bit it's not needed.